dataprofiler/profilers/unstructured_labeler_profile.py

```diff
@@ -94,16 +95,17 @@ def __add__(self, other):
 
     def report(self, remove_disabled_flag=False):
         """
-        Private abstract method for returning report.
+        Return report.
```

Comment:
this technically returns the `self.profile` object? Even thought we have this called `report()` we are returning a `profile` object?

Reply:
The previous docstring also said that it returns a report, but I'll go ahead and change it if what you said is the case.

Reply:
👍